### ISSUE TITLE
'print_escaped' is deprecated but is still being used

### ISSUE THREAD

**Issue by JoelleFV** [JuliaGraphics/Tk.jl#135]
When I simply do :
`using Tk`

I get :
```
WARNING: print_escaped is deprecated, use escape_string instead.

 in depwarn(::String, ::Symbol) at .\deprecated.jl:64
 in print_escaped(::Base.AbstractIOBuffer{
Array{UInt8,1}}, ::Vararg{Any,N}) at .\deprecated.jl:30
 in init() at C:\Users\Jowelz\.julia\v0.5\Tk\src\tkwidget.jl:49
 in include_from_node1(::String) at .\loading.jl:488 (repeats 2 times)
 in eval(::Module, ::Any) at .\boot.jl:234
 in require(::Symbol) at .\loading.jl:415
 in include_string(::String, ::String) at .\loading.jl:441
 in eval(::Module, ::Any) at .\boot.jl:234
 in (::Atom.##65#68)() at C:\Users\Jowelz\.julia\v0.5\Atom\src\eval.jl:102
 in withpath(::Atom.##65#68, ::Void) at C:\Users\Jowelz\.julia\v0.5\CodeTools\src\utils.jl:30
 in withpath(::Function, ::Void) at C:\Users\Jowelz\.julia\v0.5\Atom\src\eval.jl:38
 in macro expansion at C:\Users\Jowelz\.julia\v0.5\Atom\src\eval.jl:101 [inlined]
 in (::Atom.##64#67{Dict{String,Any}})() at .\task.jl:60
while loading C:\Users\Jowelz\.julia\v0.5\Tk\src\tkwidget.jl, in expression starting on line 457
WARNING: print_escaped is deprecated, use escape_string instead.
 in depwarn(::String, ::Symbol) at .\deprecated.jl:64
 in print_escaped(::Base.AbstractIOBuffer{Array{UInt8,1}}, ::Vararg{Any,N}) at .\deprecated.jl:30
 in init() at C:\Users\Jowelz\.julia\v0.5\Tk\src\tkwidget.jl:53
 in include_from_node1(::String) at .\loading.jl:488 (repeats 2 times)
 in eval(::Module, ::Any) at .\boot.jl:234
 in require(::Symbol) at .\loading.jl:415
 in include_string(::String, ::String) at .\loading.jl:441
 in eval(::Module, ::Any) at .\boot.jl:234
 in (::Atom.##65#68)() at C:\Users\Jowelz\.julia\v0.5\Atom\src\eval.jl:102
 in withpath(::Atom.##65#68, ::Void) at C:\Users\Jowelz\.julia\v0.5\CodeTools\src\utils.jl:30
 in withpath(::Function, ::Void) at C:\Users\Jowelz\.julia\v0.5\Atom\src\eval.jl:38
 in macro expansion at C:\Users\Jowelz\.julia\v0.5\Atom\src\eval.jl:101 [inlined]
 in (::Atom.##64#67{Dict{String,Any}})() at .\task.jl:60
while loading C:\Users\Jowelz\.julia\v0.5\Tk\src\tkwidget.jl, in expression starting on line 457
```

so apparently `print_escaped` has not been replaced in Tk.jl since its deprecation in v0.5 (if I am not mistaken). I'm just an everyday user of Julia, so I will leave this to someone who knows how to do it. When taking a look at [#14052](https://github.com/JuliaLang/julia/issues/14052), it didn't seem to be a big deal to deprecate this function, so I imagine changing it would be simple.

I use v0.5.1.

Sorry if this is a duplicate or if it is meant to be that way. I looked for other posts related to this and did not find anything.

**Comment by DrKrar** [comment]
This PR [change `print_escaped` to `escape_string` ](https://github.com/JuliaGraphics/Tk.jl/pull/136) is  #136  address this.

**Comment by aviks** [comment]
Fixed by #147
